cpp/src/arrow/compute/kernels/scalar_round_arithmetic_test.cc

```diff
@@ -113,7 +112,7 @@ class TestBaseUnaryRoundArithmetic : public ::testing::Test {
   // (Array, Array)
   void AssertUnaryOp(UnaryFunction func, const std::shared_ptr<Array>& arg,
                      const std::shared_ptr<Array>& expected) {
-    ASSERT_OK_AND_ASSIGN(auto actual, func(arg, options_, nullptr));
+    ASSERT_OK_AND_ASSIGN(auto actual, func(arg, options_, nullptr))
```

Comment:
While this does compile and work we try and add a `;` to the end of these kinds of macros anyways for readability
```suggestion
    ASSERT_OK_AND_ASSIGN(auto actual, func(arg, options_, nullptr));
```

Reply:
Another clion special.  I'll turn off the option warning about empty statements.